package.json

```diff
@@ -36,7 +36,7 @@
     "verbose": true
   },
   "scripts": {
-    "test": "NODE_OPTIONS=--experimental-vm-modules jest",
+    "test": "cross-env NODE_OPTIONS=--experimental-vm-modules jest",
```

Comment:
I would appreciate if you could split this into a separate commit, glad it works on windows with this!